src/Pages/Index.cshtml

```diff
@@ -4,7 +4,16 @@
     ViewData["Title"] = "ePiggy";
 }
 
-
+@if(Request.Cookies.ContainsKey("recoveryCode"))
+{
+    <div class="alert alert-warning" role="alert">
+        Password recovery is in progress! <a asp-page="/forgotPassword" class="alert-link">Go to recovery</a> to finish it!.
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+   
+}
```

Comment:
Not sure if this is supposed to work this way, but if you start password recovery, and then login without finishing it, this popup doesn't appear

Reply:
Yup, it shouldn't since it resets recover if logged in.